### DISCUSSION TITLE
text extraction, page rendering

### DISCUSSION THREAD

**Discussion by mark-summerfield** [pdfcpu/pdfcpu#580] [Q&A] [answered]
I have two questions:

1. Does pdfcpu support text extraction so that for a given pdf and page it will give me a slice containing every word (or character) on the page along with the rectangle it occupies (and ideally its font and color)? Use cases: cli grep; gui hightlighting of found words, pdf to text conversion.
2. Does pdfcpu support page rendering, e.g., for a given pdf and page can it render that page to say svg or png format?

**Comment by hhrutter** [comment, marked as answer]
1) not right now but in the pipeline #122
2) out of scope  #45
